components/BridgeContent/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useCallback, useEffect, useState, memo } from 'react';
+import SocketPlugIn from 'components/BridgeContent/SocketPlugIn';
+import { Box, Typography } from '@mui/material';
+import { t } from 'i18next';
+import { ActiveRoute } from 'types/Bridge';
+import { useWeb3 } from 'hooks/useWeb3';
+import { fetchActiveRoutes } from './utils';
+import SocketTxHistory from './SocketTxHistory';
+
+const BridgeContent = () => {
+  const [activeRoutes, setActiveRoutes] = useState<ActiveRoute[] | undefined>();
+  const { walletAddress } = useWeb3();
+
+  const updateRutes = useCallback(async () => {
+    if (!walletAddress) return;
+    const routes = await fetchActiveRoutes(walletAddress);
+    setActiveRoutes(routes);
+  }, [walletAddress]);
+
+  useEffect(() => {
+    updateRutes();
+  }, [updateRutes, walletAddress]);
+
+  return (
+    <Box
+      sx={{ display: 'flex', justifyContent: 'center' }}
+      flexDirection={'column'}
+      mx={'auto'}
+      mt={'40px'}
```

Comment:
For paddings and margins we use "8px" = 1 unit in mui
So, this should be `mt={5}`